static/app/components/dropdownMenuItem.tsx

```diff
@@ -199,7 +194,14 @@ const MenuItem = ({
 
   return (
     <MenuListItem
-      ref={ref}
+      ref={el => {
```

Comment:
It looks like there is a `utils/mergeRefs.tsx` that you can use here

Reply:
Oh woah that's neat! Updated in https://github.com/getsentry/sentry/pull/39800/commits/5920587235261208eaf10bd6d2b511ccd3145fe3